frontend/views/containers/access/LoginModal.vue

```diff
@@ -1,5 +1,5 @@
 <template lang='pug'>
-  modal-template(class='has-background' ref='modal')
+  modal-template(class='has-background' ref='modal' :a11yTitle='L("Login")')
```

Comment:
Andrea noticed that we've been writing "Login" incorrectly, it should be "Log In"